pages/index.js

```diff
@@ -20,7 +22,10 @@ export default function Home({userName, handleUserNameFormSubmit}) {
       {userName === "DontRender" ? (
         ""
       ) : (
-        <Navigation selected={"index"} userName={userName}></Navigation>
+        <Navigation selected={"index"} userName={userName}>
+          {" "}
```

Comment:
Not sure but do you need this space here?

Reply:
no, I dont. that is the linter. I havent found the right settings to turn it off.